### PR TITLE
icon loop and aspect

### PR DESCRIPTION
**Pull request by travismeyer**
Adds a looped map for generating classes for icons, as well as a mixin for applying aspect ratio with pseudo selector

### CODE REVIEW

**Comment by kyleshevlin** [comment]
I would wrap the loop in a mixin and add it to the fine_mixins file if we are going to use this. Then you could have an _icons.scss file and it would basically have one line of `@include iconloop;`

As for the utility, I think this would be more useful if we were hand rolling the icons in each project, but I've had little trouble with designers changing icons and/or using Icomoon to handle this. I'd rather just copy/paste Icomoon and then make updates when necessary.

File: source/stylesheets/globals/imports/finescss/_fine_mixins.scss [hunk 1]
```diff
@@ -291,3 +291,36 @@
...
+/// to a ratio and the screen size
+/// @param {number} $width
+/// @param {number} $height
+/// @example scss - aspect-ratio mixin
+///  @include aspect-ratio(1, 0.8);
+@mixin aspect-ratio($x, $y) {
+
+  @if (type-of($x) != number) or
+      (type-of($y) != number) {
+    @error 'You need to enter a number as the second and third paramaters.'
```
Comment (kyleshevlin) [inline source/stylesheets/globals/imports/finescss/_fine_mixins.scss]: Second and Third? there are only two params here.

**Comment by kyleshevlin** [comment]
Aspect ratio works for me, I've hand rolled something similar a few times.

File: source/stylesheets/globals/imports/finescss/_fine_mixins.scss (hunk 1 above)
Comment (travismeyer) [inline source/stylesheets/globals/imports/finescss/_fine_mixins.scss]: woops, relic from when this mixin included a `position` parameter

**Pull request merged**